src/index.js

```diff
@@ -20,13 +20,14 @@ function encodeAttr(str) {
 	return (str+'').replace(/"/g, '&quot;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
 }
 
+let links = {};
```

Comment:
I think one issue with the global version is that it won't get cleaned after each outer `parse()` call.  So links declared in one document would be available as references in another.

Are you able to push the alternate contextual version?